src/App.js

```diff
@@ -9,6 +9,7 @@ export default function App() {
       <Route path="/video/:videoId" element={<SingleVideo />} />
       <Route path="/liked" element={<Liked />} />
       <Route path="/watchlater" element={<WatchLater />} />
+      <Route path="/history" element={<History />} />
     </Routes>
```

Comment:
you can use seprate folder route for this

Reply:
ok will do that